zero_bin/tools/prove_rpc.sh

```diff
@@ -13,10 +13,6 @@
 export RUST_MIN_STACK=33554432
 export RUST_BACKTRACE=1
 export RUST_LOG=info
-# Disable the lld linker for now, as it's causing issues with the linkme package.
-# https://github.com/rust-lang/rust/pull/124129
-# https://github.com/dtolnay/linkme/pull/88
-export RUSTFLAGS='-C target-cpu=native -Zlinker-features=-lld'
```

Comment:
I see you moved the disabling of lld linker in `config.toml` but why removing `-C target-cpu=native`. This should be disabled when building cross-platform images, but it doesn't hurt keeping it in the scripts.

Reply:
I'll take your lead on this